src/mem/metaslab.h

```diff
@@ -177,6 +177,16 @@ namespace snmalloc
   public:
     constexpr MetaEntry() = default;
 
+    /**
+     * Constructor, provides the remote and sizeclass embedded in a single
+     * pointer-sized word.  This format is not guaranteed to be stable and so
+     * the second argument of this must always be the return value from
+     * `get_remote_and_sizeclass`.
+     */
+    MetaEntry(Metaslab* meta, uintptr_t remote_and_sizeclass)
+    : meta(meta), remote_and_sizeclass(remote_and_sizeclass)
+    {}
```

Comment:
Is this used?  (is github just hiding a file from me in the diff view?  I don't think so?)

Reply:
[Here](https://github.com/microsoft/verona/blob/4cb68fbae688c1e0ef8b701e61eed00d072a6374/experiments/process_sandbox/src/libsandbox.cc#L148).